samples/Microsoft.AspNetCore.Proxy.Samples/web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <system.webServer>
+    <handlers>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
+    </handlers>
+    <aspNetCore processPath="./Microsoft.AspNetCore.Proxy.Samples.exe" arguments="" forwardWindowsAuthToken="false" />
```

Comment:
FIled this https://github.com/aspnet/IISIntegration/issues/127 5 minutes ago. Maybe forward slashes work for current  folder?